docs/community/journey.md

```diff
@@ -0,0 +1,82 @@
+
+* The gas feature is not implemented yet, nor is native value transfer.
+
+* There is no streaming of events. This will limit how your UI updates based on events being listened to.
+
+* Testnet might crash unexpectedly as sharp edges are smoothed away and edge cases fixed.
+
+* Expect to lose all data when Testnet crashes or is restarted. 
+
+## Challenges
+
+Below are a set of challenges to get you started. For each challenge you successfully complete, you'll be rewarded task points which get transformed into OBX tokens as per the [Tokenomics](https://github.com/obscuronet/obscuro-project/wiki/Tokenomics)
+
+### Warmup - 20 points
+Get set up and deploy a simple contract.
+* Configure the Wallet extension and Metamask following the instructions [here](https://docs.obscu.ro/wallet-extension/wallet-extension.html)
+* Try deploying any contract (e.g. the default Storage contract). Remember to select Injected Provider as your Environment ![Step 1 screenshot](https://images.tango.us/public/screenshot_017a4a1c-b655-4eda-8c6e-7a298e1faa70.png?crop=focalpoint&fit=crop&fp-x=0.0945&fp-y=0.1827&fp-z=2.5115&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=2027%3A1103)
+
+### Deploy the Guessing game - 20 points
+The guessing game is a very basic contract that showcases some of Obscuro's capabilities. The goal of the same is to simply guess the secret number the game has selected. Each time an attempt is made, an entrance fee of 1 token is paid. If a player guesses correctly, they win the entire pot and a new random number is selected.
+
+Before we get into how the game works, let's see it in action and play. See the instructions [here](https://docs.obscu.ro/testnet/example-dapps.html)
+
+Now that we've played the game let's try deploying the Guessing game to the Obscuro testnet. You can clone the repo from [here](https://github.com/obscuronet/number-guessing-game). 
+
+#### How the Guessing game works
+There are two main functions of interest. First, the *attempt* function allows a player to guess the number and check if they've correctly guessed the secret number. If they have, the prize pot is paid out to them.
+![attempt function](../../assets/images/guessing.png)
+
+The other is the *setNewTarget* function. This checks the game is in a state ready to be reset and then sets a secret number based on various block parameters.
+![setNewTarget function](../../assets/images/setnewtarget.png)
+
+### Extend the Guessing game with a proper win notification - 50 points
+You'll have noticed immediately that on winning the game, there is no notification! The only thing that happens is the game transfers to you all the tokens in the pot. 
+
+The challenge here is to extend the game with events (or anything else) to inform the winning player that they've won and how much they've won. 
```

Comment:
Do you want to mention anything about implementing a good UI for this?

Reply:
Good point